packages/typespec-autorest-canonical/package.json

```diff
@@ -0,0 +1,88 @@
+{
+  "name": "@azure-tools/typespec-autorest-canonical",
+  "version": "0.1.0",
+  "author": "Microsoft Corporation",
+  "description": "TypeSpec library for emitting canonical swagger",
+  "homepage": "https://azure.github.io/typespec-azure",
+  "readme": "https://github.com/Azure/typespec-azure/blob/main/README.md",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Azure/typespec-azure.git"
+  },
+  "bugs": {
+    "url": "https://github.com/Azure/typespec-azure/issues"
+  },
+  "keywords": [
+    "typespec",
+    "autorestcanonical"
+  ],
+  "type": "module",
+  "main": "dist/src/index.js",
+  "tspMain": "lib/autorest-canonical.tsp",
+  "exports": {
+    ".": {
+      "types": "./dist/src/index.d.ts",
+      "default": "./dist/src/index.js"
+    },
+    "./testing": {
+      "types": "./dist/src/testing/index.d.ts",
+      "default": "./dist/src/testing/index.js"
+    }
+  },
+  "engines": {
+    "node": ">=18.0.0"
+  },
+  "scripts": {
+    "clean": "rimraf ./dist ./temp",
+    "build": "npm run regen-autorest-canonical-openapi-schema && tsc -p . && npm run lint-typespec-library",
+    "watch": "tsc -p . --watch",
+    "lint-typespec-library": "tsp compile . --warn-as-error --import @typespec/library-linter --no-emit",
+    "regen-autorest-canonical-openapi-schema": "tsp compile ./schema/autorest-canonical-openapi-schema.tsp --warn-as-error && node ./.scripts/schema-json-to-js.js",
+    "test": "vitest run",
+    "test:watch": "vitest -w",
+    "test:ui": "vitest --ui",
+    "test-official": "vitest run --coverage --reporter=junit --reporter=default --no-file-parallelism",
+    "lint": "eslint . --ext .ts --max-warnings=0",
+    "lint:fix": "eslint . --fix --ext .ts",
+    "regen-docs": "tspd doc .  --enable-experimental  --output-dir ../../docs/emitters/typespec-autorest-canonical/reference"
+  },
+  "files": [
+    "lib/*.tsp",
+    "schema/dist/schema.js",
+    "dist/**",
+    "!dist/test/**"
+  ],
+  "peerDependencies": {
+    "@azure-tools/typespec-azure-core": "workspace:~",
+    "@azure-tools/typespec-client-generator-core": "workspace:~",
+    "@typespec/compiler": "workspace:~",
+    "@typespec/http": "workspace:~",
+    "@typespec/openapi": "workspace:~",
+    "@typespec/rest": "workspace:~",
+    "@typespec/versioning": "workspace:~"
```

Comment:
Do you actually need versioning if you are going to ignore it?

Reply:
we need the api to get informations like included versions, and renamedfrom, et al. so need to have it.